samples/subsys/zbus/uart_bridge/src/core.c

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) 2022 Rodrigo Peixoto <rodrigopex@gmail.com>
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#include "messages.h"
+
+#include <zephyr/logging/log.h>
+#include <zephyr/zbus/zbus.h>
+LOG_MODULE_DECLARE(zbus, CONFIG_ZBUS_LOG_LEVEL);
+
+ZBUS_CHAN_DECLARE(start_measurement_chan);
+
+ZBUS_CHAN_DEFINE(version_chan,	     /* Name */
+		 true,		     /* Read only */
+		 struct version_msg, /* Message type */
+
+		 NULL,		       /* Validator */
+		 ZBUS_OBSERVERS_EMPTY, /* observers */
+		 ZBUS_MSG_INIT(.major = 0, .minor = 1,
+			       .build = 1023) /* Initial value major 0, minor 1, build 1023 */
+);
+
+void core_thread(void)
+{
+	struct action_msg start = {false};
+
+	while (1) {
+		LOG_DBG("Core sending start measurement with status %d", start.status);
```

Comment:
Is this correct? You say sending with status.status but then you invert it and send, maybe you should send and then invert?